native/core/src/execution/jni_api.rs

```diff
@@ -265,14 +259,33 @@ fn parse_bool(conf: &HashMap<String, String>, name: &str) -> CometResult<bool> {
 }
 
 /// Prepares arrow arrays for output.
-fn prepare_output(
+unsafe fn prepare_output(
```

Comment:
Is it possible to narrow down the area of `unsafe`?

Reply:
Okay